cross/flac/patches/flac-1.3.2-remove-cflags-wextra.patch

```diff
@@ -0,0 +1,13 @@
+--- configure.orig	2017-02-15 06:56:50.092654078 +0100
++++ configure	2017-02-15 06:56:59.596796743 +0100
+@@ -20571,8 +20571,8 @@
+ 
+ 
+ if test x$ac_cv_c_compiler_gnu = xyes ; then
+-	CFLAGS="$CFLAGS -Wall -Wextra -Wstrict-prototypes -Wmissing-prototypes -Waggregate-return -Wcast-align -Wnested-externs -Wshadow -Wundef -Wmissing-declarations -Winline " # -Wcast-qual -Wbad-function-cast -Wwrite-strings -Wconversion
+-	CXXFLAGS="$CXXFLAGS -Wall -Wextra -Wcast-align -Wshadow -Wwrite-strings -Wctor-dtor-privacy -Wnon-virtual-dtor -Wreorder -Wsign-promo -Wundef " # -Wcast-qual -Wbad-function-cast -Wwrite-strings -Woverloaded-virtual -Wmissing-declarations
```

Comment:
Removing `-Wextra` CC flag is required for following archs:
* evansport-4.3
* ppc824x-4.2
* evansport-5.1
* powerpc-4.0
* comcerto2k-5.1
* comcerto2k-5.0
* evansport-5.0

I have found no easy way to apply this patch conditionnaly